src/app/core/cms/haldoc.ts

```diff
@@ -108,11 +108,11 @@ export class HalDoc {
     }
   }
 
-  has(rel: string): boolean {
+  has(rel: string, mustBeList = false): boolean {
```

Comment:
There's still one instance in CMS (the audio-version's `_embedded` audio-files) where the contents aren't behind a `prx:items` array.  And I needed the ability to check for that, for now.